packages/fx-core/src/plugins/solution/fx-solution/arm.ts

```diff
@@ -1083,16 +1083,16 @@ async function compileBicepToJson(
 
 // Context used by handlebars to render the main.bicep file
 export class ArmTemplateRenderContext {
-  public Plugins: string[];
-  public PluginOutput: { [PluginName: string]: PluginOutputContext };
+  public Plugins: Record<string, PluginContext> = {};
```

Comment:
Seems context does not need to have the "Plugins" level.
```
{
    'fx-resource-bot': {
        Provision: {},
        Configuration: {},
        References: {}
    }
}
```

If remove the 'Plugins' layer, bicep file can be more concise like: 
```
module apimProvision '{{fx-resource-apim.Provision.apim.ProvisionPath}}'  = {
...
}
```
instead of `{{Plugins.fx-resource-apim.Provision.apim.ProvisionPath}}`

Reply:
sure